yt_dlp/extractor/mojevideo.py

```diff
@@ -0,0 +1,43 @@
+import re
+from .common import InfoExtractor
+from ..utils import ExtractorError
+
+
+class MojevideoIE(InfoExtractor):
+    _VALID_URL = r'https://www\.mojevideo\.sk/video/(?P<id>[0-9]+)'
```

Comment:
`[0-9]+` is incorrect? and do we want to make the `www` subdomain optional?
```suggestion
    _VALID_URL = r'https?://(www\.)?mojevideo\.sk/video/(?P<id>\w+)'
```